packages/ui/jest-setup.ts

```diff
@@ -29,3 +30,12 @@ const fetchMock = jest.spyOn(window, 'fetch');
 beforeEach(() => {
   fetchMock.mockResolvedValue(null as unknown as Response);
 });
+
+beforeAll(async () => {
```

Comment:
This works for running tests, but `yarn build` fails saying those variables are not defined.

Reply:
Let's take a look :ok: